src/mlpack/core/data/load.hpp

```diff
@@ -21,6 +21,7 @@
 #include "format.hpp"
 #include "dataset_mapper.hpp"
 #include "image_info.hpp"
+#include "csv_parser.hpp"
```

Comment:
You need to include the `load_csv.hpp` here in order to avoid the error.

Reply:
I did, still didn't work and also `load_csv.hpp` is included in `load_impl.hpp` anyways, so shouldn't that be enough?

Reply:
@shrit It's still not working, maybe it's a circular include problem, not sure, I can't test the DatasetMapper implementation without the solving this error, have a look at it if possible...